app/views/api_keys/show.html.haml

```diff
@@ -0,0 +1,14 @@
+.container
+  .lead API Key
+
+  %p Your can access BIP API using the following key:
+
+  .row
+    .col-sm-6
+      %code
+        = @api_key.token
+
+    .col-sm-6
+      = button_to "Renew", recreate_api_keys_path, method: "put", class: 'btn btn-default'
```

Comment:
Please add the following warning text:

"This operation will revoke your current API Key. That means it will no longer be valid and every API call using that API Key value will fail. When you decide to renew your API Key, please make sure that you use the new value in your BIP client."